setup.py

```diff
@@ -0,0 +1,96 @@
+"""
+
+Setup Script for RedditVideoMakerBot
+
+"""
+
+# Imports
+import os
+import time
+from utils.console import print_markdown
+from utils.console import print_step
+from utils.console import print_substep
+from rich.console import Console
+from utils.loader import Loader
+console = Console()
+
+# These lines ensure the user:
+# - knows they are in setup mode
+# - knows that they are about to erase any other setup files/data.
+
+print_step("Setup Assistant")
+
+print_markdown(
+	"### You're in the setup wizard. Ensure you're supposed to be here, then type yes to continue. If you're not sure, type no to quit."
+)
+
+# This Input is used to ensure the user is sure they want to continue.
+ensureSetupIsRequired = input("Are you sure you want to continue? > ")
+if ensureSetupIsRequired != "yes":
+	console.print("[red]Exiting...")
+	time.sleep(0.5)
+	exit()
+else:
+	# Again, let them know they are about to erase all other setup data.
+	console.print("[bold red] This will overwrite your current settings. Are you sure you want to continue? [bold green]yes/no")
+	overwriteSettings = input("Are you sure you want to continue? > ")
```

Comment:
I would all .casefold() to all of the input checks as somebody might type 'Yes' and it would fail

Reply:
agreed

Reply:
👍